### PR TITLE
Layout changes

### PR DESCRIPTION
**Pull request by pavpanchekha**

### CODE REVIEW

File: book/layout.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ and the HTML element below it), but a tree nonetheless!
 
 By the way, since we now have `DocumentLayout`, let's rename `Layout`
 so it's less ambiguous. I like `BlockLayout` as a name, because we
-ultimately want `Layout` to represent a block of text, like a
+ultimately want it to represent a block of text, like a
```
Comment (pavpanchekha) [inline book/layout.md:121]: Using `Layout` here is weird because we're talking about renaming it.

File: book/layout.md [hunk 2]
```diff
@@ -188,7 +188,20 @@ The core idea is that we'll have a whole tree of `BlockLayout` objects
 that contain text, and they'll lay out their contents the way we've
 already implemented. But there will also be new, intermediate
 `BlockLayout`s with `BlockLayout` children, and they will stack their
-children vertically.
+children vertically. (An example is shown in Figure 1.)
```
Comment (pavpanchekha) [inline book/layout.md:191]: This figure and parenthetical is moved from much later in the chapter.

File: book/layout.md [hunk 3]
```diff
@@ -671,7 +671,7 @@ to draw, but only in inline mode:[^why-not-change]
 [^why-not-change]: Why not change the `display_list` field inside a
 `BlockLayout` to contain `DrawText` commands directly? I suppose you
 could, but I think it's cleaner to create all of the draw
-commands in one place.
+commands in `paint`.
```
Comment (pavpanchekha) [inline book/layout.md:674]: It's more than one `paint` definition so "one place" is a bit weird.

File: book/layout.md [hunk 4]
```diff
@@ -702,7 +702,7 @@ background has to be drawn *below* that text. Note also that `paint_tree`
 calls `paint` before recursing into the subtree, so the subtree also paints
 on top of this background, as desired.
 
-With the display list filled out, we need to `draw`
+With the display list filled out, we need to draw
```
Comment (pavpanchekha) [inline book/layout.md:705]: I think we used to call the method itself `draw` but we don't anymore, so this isn't a method reference. The Tk functions we call are all `create_xxx` so `draw` doesn't really make sense.

File: book/layout.md [hunk 5]
```diff
@@ -770,14 +770,14 @@ def scrolldown(self, e):
 ```
 
 Note the `2*VSTEP`, to account for a `VSTEP` of whitespace at the top
-and bottom of the page.
+and bottom of the page. With layout the [browser.engineering
+homepage](https://browser.engineering/) now looks a bit better---see
+Figure 3.
```
Comment (pavpanchekha) [inline book/layout.md:775]: Moved from next paragraph where it is a bit weird.

File: book/layout.md (hunk 2 above)
Comment (chrishtr) [reply book/layout.md:191]: How important is this?

**Review by chrishtr — APPROVED** [review]

**Pull request merged**